packages/@tailwindcss-node/src/compile.ts

```diff
@@ -87,6 +87,13 @@ export async function loadModule(id: string, base: string, onDependency: (path:
       throw new Error(`Could not resolve '${id}' from '${base}'`)
     }
 
+    if (typeof resolvedPath !== 'string') {
+      return {
+        base,
+        module: resolvedPath,
+      }
+    }
```

Comment:
I know _why_ this is done this way but it feels super wrong. Can we have a `__tw_load` function or something instead?